src/core/manager.c

```diff
@@ -877,6 +877,19 @@ enum {
         _GC_OFFSET_MAX
 };
 
+static void unit_gc_mark_good(Unit *u, unsigned gc_marker)
+{
```

Comment:
opening bracket on same line as the function name please, please see CODING_STYLE.